### DISCUSSION TITLE
Revisit/Update how we explain Highest Imputation (vignette and function documentation)

### DISCUSSION THREAD

**Discussion by bms63** [pharmaverse/admiral#2090] [General]
This comes out of the https://github.com/pharmaverse/admiral/issues/2028 where the imputation vignette has discussion stating that admiral does not allow a user to preserve parts of the partial dates.  This is not true and we decided to remove this section from the vignette.

In the PR to address this issue a **_poor attempt_** was made to better explain highest imputation.  I found working through the examples to be the best way to understand H.I., but trying to explain it in a pithy and concise way was a bit of a struggle.  I do believe a user can grasp the H.I. and the date/time functions by working from the examples as they are scaffold really well...but I think the documentation is hard to follow.

I think we should discuss this issue here and in upcoming meetings and then update how we explain H.I.  I was leaning towards a diagram/flowchart as the max/min, date/time and preserve arguments all forms part of the discussion around H.I.!  

Also...pointed about by @zdz2101 that in every release we tend to have a bug around out `dtm/dt` functions

Some relevant links:

https://github.com/pharmaverse/admiral/pull/2088#discussion_r1320183370
https://pharmaverse.github.io/admiral/cran-release/reference/derive_vars_dtm.html

@pharmaverse/admiral

**Comment by manciniedoardo** [comment]
Discussed - we shall close as no user feedback that this is required.
